esmvaltool/diag_scripts/MyDiag_test_ta.ncl

```diff
@@ -55,6 +55,7 @@
 load "$ESMValTool_interface_data/ncl.interface"
 
 ; Auxiliary NCL routines
+load "./interface_scripts/interface.ncl"
```

Comment:
Shouldn't this replace line 55?
Also applies to other diags

Reply:
Not at the moment: `./interface_scripts/interface.ncl` contains the part of the ncl template with code that never changed, while `$ESMValTool_interface_data/ncl.interface` contains the data that was filled. However, the plan is to pass the file containing the configuration data as the first argument to the diagnostic (see Niels' first comment in this issue https://github.com/ESMValGroup/ESMValTool/issues/15), so I think the line `load "$ESMValTool_interface_data/ncl.interface"` will change a bit in the future.

Reply:
Does this still work as expected? Or should `./interface_scripts/interface.ncl` also contain the line `./interface_scripts/interface.ncl`?

Reply:
Seems to work fine